cmd/cloudshell_open/appfile.go

```diff
@@ -44,8 +44,9 @@ type hook struct {
 }
 
 type hooks struct {
-	PreCreate hook `json:"precreate"`
+	PreCreate  hook `json:"precreate"`
 	PostCreate hook `json:"postcreate"`
+	Build      hook `json:"build"`
```

Comment:
I think this should be called PostBuild or PreBuild.
More verbose the better.

Reply:
Wait, I think I got this wrong:

Does this override the build method (i.e. docker jib etc)?
If so:
- I think this is technically not a hook, but more like an "override". If - how does it know what image name to build as?

Please add README.md docs as well.

Reply:
Yeah, more details in https://github.com/GoogleCloudPlatform/cloud-run-button/issues/85#issuecomment-599804703

I added docs and two new env vars (image url and app dir).